File: Software Technologies/17_Exam_Prep/Log_Noziroh/JavaScript/models/Report.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Report = sequelize.define('Report', {
        status:{
            type: Sequelize.TEXT,
            allowNull: false
        },

        message:{
            type: Sequelize.TEXT,
            allowNull: false
        },

        origin:{
            type: Sequelize.TEXT,
                allowNull: false
        }
    },{
        timestamps: false,
    });

    return Report;
};

File: Software Technologies/17_Exam_Prep/Log_Noziroh/C#/LogNoziroh/Models/Report.cs
namespace LogNoziroh.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Report
    {
        [Key]
        public int Id { get; set; }

        public string Status { get; set; }

        public string Message { get; set; }

        public string Origin { get; set; }
    }
}
